include/dev/INA138.hpp

```diff
@@ -0,0 +1,43 @@
+#ifndef LVSS_INA138_HPP
+#define LVSS_INA138_HPP
+
+#include "EVT/io/ADC.hpp"
+#include "EVT/utils/log.hpp"
+
+namespace IO = EVT::core::IO;
+namespace DEV = EVT::core::DEV;
+namespace log = EVT::core::log;
+
+namespace LVSS {
+
+/**
+ * Class for LVSS current sensor feature
+ */
+class INA138 {
+public:
+    /**
+      * Constructor for current sensing class
+      */
+    INA138(IO::ADC& adc0);
+
+    /**
+      * * Get the current detected by the INA
+      *
+      * @return The current
+      */
+    uint32_t readCurrent(IO::ADC&);
```

Comment:
You should use the instance variable here instead of taking the ADC as an argument
```suggestion
    uint32_t readCurrent();
```